accounts.go

```diff
@@ -49,6 +49,22 @@ func (lw *LibWallet) GetAccountsRaw(requiredConfirmations int32) (*Accounts, err
 	}, nil
 }
 
+func (accounts *Accounts) GetAccountIterator() *AccountIterator {
```

Comment:
The following reads better as `accounts.Iterator` rather than `accounts.GetAccountIterator`:
```suggestion
func (accounts *Accounts) Iterator() *AccountIterator {
```